.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- HTML Comments can be left in place or removed. -->
+
+Proposed changes
+======
+
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. -->
+
+Types of changes
+======
+
+<!-- What types of changes does your code introduce to Rancher? Bugfix, New Feature, Breaking Change, etc -->
+
+Verification
+======
+
+<!-- How can the changes be verified? Please provide whatever additional information necessary to help verify the proposed changes. -->
+
+Linked Issues
+======
+
+<!-- Link any related issues, pull-requests, or commit hashes that are relevant to this pull request. If you are opening a PR without a corresponding issue please consider creating one first, [here](https://github.com/rancher/k3s/issues). A functional example will greatly help QA with verifying/reproducing a bug or testing new features. -->
```

Comment:
```suggestion
<!-- Link any related issues, pull-requests, or commit hashes that are relevant to this pull request. If you are opening a PR without a corresponding issue please consider creating one first, [here](https://github.com/rancher/rke2/issues). A functional example will greatly help QA with verifying/reproducing a bug or testing new features. -->
```

Reply:
I tweaked that a bit, since markdown doesn't work in comments.